src/fides/ctl/core/export.py

```diff
@@ -232,7 +232,7 @@ def generate_system_records(
                 data_protection_impact_assessment["link"],
             ]
             num_privacy_declaration_fields = 12
-            privacy_declaration_start_index = 6
+            privacy_declaration_start_index = 7
```

Comment:
perhaps your idea to explicitly write out all the N/A's would've been better 😅

Reply:
We still can!